### PR TITLE
fix(string): escape pattern toString

### PR DESCRIPTION
**Pull request by intech**
Issue #221

### CODE REVIEW

**Comment by intech** [comment]
Снимок экрана от 2021-03-09 22-35-50
I tested the bug on my computer and passed all tests successfully.

**Comment by intech** [comment]
@erfanium what is the best name for it? Or replace the `quote` test?

**Review by erfanium — APPROVED** [review]

**Comment by icebob** [comment]
@erfanium please wait for my approving before merge.

**Pull request merged**
